### PR TITLE
cc17

### PR DESCRIPTION
**Pull request by ZenAtomsk**
- [ ] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/queueWithStacks/queue-with-stacks.js [hunk 1]
```diff
@@ -45,4 +45,6 @@ class PseudoQueue{
   }
 
   // dequeue()
-}
\ No newline at end of file
+}
+
+module.exports = Queue;
```
Comment (adrianhuebner) [inline javascript/code-challenges/queueWithStacks/queue-with-stacks.js:50]: It looks like this is causing an error, I am getting back: `ReferenceError: Queue is not defined`

Comment (adrianhuebner) [inline javascript/code-challenges/queueWithStacks/queue-with-stacks.js:50]: It should be PseudQueue
